modules/macos_mobileconfig_profiles/manifests/desktop_background.pp

```diff
@@ -0,0 +1,11 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class macos_mobileconfig_profiles::desktop_background {
+
+    mac_profiles_handler::manage { 'com.github.erikberglund.ProfileCreator.CB0ADD7C-78DE-4C29-854B-3FC332E953A6':
+        ensure      => 'present',
+        file_source => 'puppet:///modules/macos_mobileconfig_profiles/org.mozilla.desktop_background.mobileconfig',
```

Comment:
Is there another format for the profiles that could show the settings? I trust these, but I'm concerned we don't have a way to see what the profiles do when looking at them in the repo.

![image](https://user-images.githubusercontent.com/5540936/55527930-45393a80-5658-11e9-882b-d602c5d380e5.png)

Reply:
I'm not sure if there is a cli tool to inspect them although they are binary files, you can open them up and see the payload.  The reason they are binaries is that I'm creating them the ProfileCreator tool and then signing them with our Developer Installer cert.